nemo_skills/pipeline/utils/mounts.py

```
@@ -118,6 +129,16 @@ def check_mounts(
         # Just check if the paths are mounted
         for path in mount_map.keys():
             if path is not None:
+                if "${" in path and "}" in path:
+                    path = os.path.expandvars(path)
+
+                    # Replace ${VAR} with the value from env_vars
+                    if "$" in path:
+                        raise ValueError(
+                            f"Cluster config item `{path}` contains env variable placeholders, but the required env var is "
+                            f"not provided in the cluster config or environment to resolve."
+                        )
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)